.github/workflows/publish-containers.yml

```diff
@@ -36,9 +55,11 @@ jobs:
           images: |
             ghcr.io/${{ steps.image.outputs.image_name }}
           tags: |
-            type=ref,event=branch,prefix=unstable-
+            type=ref,event=branch,prefix=${{ inputs.prefix }}
             type=pep440,pattern={{version}}
             type=pep440,pattern={{major}}.{{minor}}
+            type=raw,value=${{ inputs.version }},enable=${{ inputs.version != null }}
+          context: git
```

Comment:
we want ref and sha from git instead of action [context](https://github.com/docker/metadata-action?tab=readme-ov-file#context-input)